dandi/organize.py

```diff
@@ -307,9 +308,28 @@ def _assign_obj_id(metadata, non_unique):
     # Avoid heavy import by importing within function:
     from .pynwb_utils import get_object_id
 
-    msg = "%d out of %d paths are not unique" % (len(non_unique), len(metadata))
+    msg = "%d out of %d paths are not unique." % (len(non_unique), len(metadata))
+    non_unique_paths = sorted(non_unique)
 
-    lgr.info(msg + ". We will try adding _obj- based on crc32 of object_id")
+    # provide more information to the user
+    def get_msg(path, indent="  "):
```

Comment:
`get_msg()` never seems to be called with a custom `indent`, so why make it a parameter?

Reply:
yeah, in initial version I had different indents, will change